src/main/java/com/echobox/api/linkedin/types/posts/Post.java

```diff
@@ -235,6 +244,13 @@ public List<String> getImageURLs() {
         .collect(Collectors.toList());
   }
   
+  public List<String> getVideoURLs() {
```

Comment:
You may want to ensure that `videos` is not null to prevent a `NullPointerException`.
The same thing is happening with the `images` mapping, which, as a workaround, we're handling in the project that uses the sdk. 
(If you could do the same check for the `images` mapping, it would be appreciated :pray:)